portfolio/src/main/webapp/script.js

```diff
@@ -30,22 +30,20 @@ async function getRandomMessage() {
 async function getReviewData() {
   const response = await fetch('/data');
   const message = await response.json();
-
-  for(var i = 0; i < message.length; i++) {
-    document.getElementById('white-background').innerHTML += "<div id='review-box'> <p id='review-text'>" + message[i] + "</p></div>";
-  }
-
-  /* alternate attempt which didn't work: 
-  *
   reviewBox = document.getElementById('white-background');
+
   for(var i = 0; i < message.length; i++) {
-    newComment = document.createElement('review-text');
+    // reviewBox.innerHTML += "<div id='review-box'> <p id='review-text'>" + message[i] + "</p></div>";
```

Comment:
Any reason to keep this commented out? If so, maybe just add another comment explaining if/when you will use this line later :)